addons/mail/static/src/utils/throttle/tests/throttle_tests.js

```diff
@@ -13,14 +13,6 @@ QUnit.module('throttle_tests.js', {
     beforeEach() {
         beforeEach(this);
         this.throttles = [];
-
-        this.start = async params => {
-            const { env, widget } = await start(Object.assign({}, params, {
-                data: this.data,
-            }));
-            this.env = env;
-            this.widget = widget;
-        };
```

Comment:
I don't think you can remove that yet.
afterEach is relying on them to be set to correctly do its job